plugins/woocommerce/tests/e2e-pw/tests/merchant/create-product-brand.spec.js

```diff
@@ -0,0 +1,181 @@
+const { test, expect } = require( '@playwright/test' );
+
+test.use( { storageState: process.env.ADMINSTATE } );
+
+test.skip( 'Merchant can add brands', async ( { page } ) => {
```

Comment:
Is it intentional that this test is skipped?

Reply:
@lanej0 In v9.4, Brands will be behind a feature flag that will be disabled by default. Therefore, Brands logic is not loaded while E2E tests run, which is why we decided to skip them for now. When the feature flag is removed, we will run these tests again. 

A similar approach was followed in other cases, where features were behind a feature flag + there were tests for these features: 

1. [Disable Launch Your Store feature flag in 9.0](https://github.com/woocommerce/woocommerce/pull/48231)
2. [Enable Launch Your Store feature flag in 9.1](https://github.com/woocommerce/woocommerce/pull/48244)